tests/test_crypto.py

```diff
@@ -971,6 +971,8 @@ def test_attributes(self):
         self.assertEqual(name.commonName, "quux")
         self.assertEqual(name.CN, "quux")
 
+        assert name.OU is None
+
```

Comment:
Boy, this looks weird alongside the unittest style assertions in the rest of this test. I'm not sure what @hynek's guidance is here so I won't merge this until he sees it.